src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -630,16 +631,22 @@ def cancel_job(self, job_id: str, parent_job_id: str = None) -> None:
                 self._running_jobs[job_id]["refresh"] = is_refreshing
                 del self._running_jobs[job_id]["canceling"]
 
-    def retry_job(self, job_id: str):
-        if job_id is None:
-            raise ValueError("Job id required for retrying")
-        if job_id not in self._running_jobs:
-            raise ValueError(f"No job present with id {job_id}")
+    def retry_jobs(self, job_id_list: List[str]):
+        for job_id in job_id_list:
+            if job_id is None:
+                raise ValueError("Job id required for retrying")
```

Comment:
this check should really be done before the `job_id_list` gets to the job manager

Reply:
It may be done redundantly in both `JobComm` and `JobManager`